vkcs/mlplatform/resource_deploy.go

```diff
@@ -0,0 +1,542 @@
+					Attributes: map[string]schema.Attribute{
+						"size": schema.Int64Attribute{
+							Required:    true,
+							Description: "Size of the volume",
+						},
+						"volume_type": schema.StringAttribute{
+							Required:    true,
+							Description: "Type of the volume",
+						},
+						"name": schema.StringAttribute{
+							Computed:    true,
+							Description: "Name of the volume",
+							PlanModifiers: []planmodifier.String{
+								stringplanmodifier.UseStateForUnknown(),
+							},
+						},
+						"volume_id": schema.StringAttribute{
+							Optional:    true,
+							Computed:    true,
+							Description: "ID of the volume",
+							PlanModifiers: []planmodifier.String{
+								stringplanmodifier.UseStateForUnknown(),
+							},
+						},
+					},
+				},
+				Optional:    true,
+				Description: "Instance's data volume configuration",
+			},
+
+			"networks": schema.ListNestedAttribute{
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"ip_pool": schema.StringAttribute{
+							Optional:    true,
+							Description: "ID of the ip pool",
+						},
+						"network_id": schema.StringAttribute{
+							Required:    true,
+							Description: "ID of the network",
+						},
+					},
+				},
+				Required:    true,
+				Description: "Network configuration",
+				Validators: []validator.List{
+					listvalidator.SizeAtMost(1),
+				},
+			},
+
+			"keypair": schema.StringAttribute{
+				Optional:    true,
+				Description: "Keypair name",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+
+			// Computed fields
+
+			"created_at": schema.StringAttribute{
+				Computed:    true,
+				Description: "Creation timestamp",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+
+			"public_ip": schema.StringAttribute{
+				Computed:    true,
+				Description: "Public IP address",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+
+			"private_ip": schema.StringAttribute{
+				Computed:    true,
+				Description: "Private IP address",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+
+			"dns_name": schema.StringAttribute{
+				Computed:    true,
+				Description: "DNS name",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+
+			"region": schema.StringAttribute{
+				Optional:    true,
+				Computed:    true,
+				Description: "The `region` to fetch availability zones from, defaults to the provider's `region`.",
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+
+			"timeouts": timeouts.Attributes(ctx, timeouts.Opts{
+				Create: true,
+				Delete: true,
+			}),
+		},
+		Description: "Manages a mlplatform Deploy resource.",
```

Comment:
I think it's better to use "ML Platform" everywhere since it's service name defined in cloud.

Reply:
Rewrote